packages/block-library/src/navigation/edit.js

```diff
@@ -33,64 +34,28 @@ import BlockColorsStyleSelector from './block-colors-selector';
 import * as navIcons from './icons';
 import NavigationPlaceholder from './placeholder';
 
+const ref = createRef();
```

Comment:
The creates a single `ref` variable, but there can be multiple instances of a block, so not sure how this could work reliably.